Week3/homework/step2-6.js

```diff
@@ -1,22 +1,21 @@
 'use strict';
 
-const arr2d = [[1, 2], [3, 4], [5, 6]];
-const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
+const flattenArray2d = [[1, 2], [3, 4], [5, 6]];
+const flattenArray3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
-function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+for (let i = 0; i < flattenArray2d.length; i++) {
+  for (let z = 0; z < flattenArray2d[i].length; z++) {
+    console.log(flattenArray2d[i][z]);
+  }
 }
 
-function flattenArray3d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+for (let i = 0; i < flattenArray3d.length; i++) {
+  for (let z = 0; z < flattenArray3d[i].length; z++) {
+    for (let j = 0; j < flattenArray3d[i].length; j++) {
+      console.log(flattenArray3d[i][z][j]);
```

Comment:
Great that you found this solution. You could also investigate using recursion which will help you avoid nesting 3 loops - generally speaking the fewer loops the better!